README.md

```diff
@@ -1 +1,51 @@
-# webpack_boilerplate
+# Leaderboard with API
+
+> Welcome to my leaderboard project, this is an interactive leaderboard excercise from Microverse.
+
+![screenshot](./screenshot.png)
+
+## Built With
+
+- HTML + CSS + Bootstrap + Javascript + Webpack
+
+## Getting Started
+
+* You will not need additional instructions on installing Prerequisites.
+* In case you have node_modules on your folder Add them to .gitignore file before you a push.
+
+### Prerequisites
+
+* No prerequisites needed for this project.
+### Installation
+
+1. Clone the repo
+   ```
+   git clone git@github.com/ccobasi/leaderboard.git
+   ```
+2. Install NPM packages
+   ```
+   npm install
+   ```
```

Comment:
* I think you can still improve your readme as the previous reviewer pointed (ie instructions on how to start the project) it is important users or visitors of your repo know how to run the project